public/view_lecture_marked_entities.php

```diff
@@ -0,0 +1,127 @@
+<?php require_once(dirname(__FILE__)."/../modules/ensure_logged_in.php"); ?>
+
+<?php
+require_once "../modules/models/user.php";
+require_once "../modules/models/section.php";
+require_once "../modules/models/section_student.php";
+require_once "../common.php";
+
+$lecture_page_id = $_GET['id'];
+
+try {
+    $course_lecture = Lecture::includes("course")->where(array('id' => $lecture_page_id));
+} catch (PDOException $error) {
+    echo "<br>" . $error->getMessage();
+}
+
+
+try {
```

Comment:
what do all these changes have to do with this issue?

Reply:
hmm this is strange this has nothing to do with it indeed. This is from the course page refactor,

i fixed it